flujo/domain/blueprint/loader_parser.py

```
@@ -352,162 +390,160 @@ def _recurse(node: object, path: str) -> None:
             try:
-                if source_file:
-                    try:
-                        setattr(p, "_source_file", str(source_file))
-                    except Exception:
-                        object.__setattr__(p, "_source_file", str(source_file))
+                from flujo.exceptions import ControlFlowError
+
+                if isinstance(e, ControlFlowError):
+                    raise
             except Exception:
                 pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)